### PR TITLE
Dev to stage 

### PR DESCRIPTION
**Pull request by darwin808**
# Description

Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.

- added media in `docs/media/api-reference` 
- fix image style in content 

https://zesty-dev-website-m3rbwjxm5q-uc.a.run.app/docs/media/api-reference/manager/
## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list the simulator and android emulators used.

- [x] Manual Test
- [ ] Unit Test
- [ ] E2E Test

# Screenshots / Screen recording
Please add screenshots or recording if applicable

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by giseleblair — DISMISSED** [review]

File: src/pages/docs/media/api-reference/index.js
Comment (giseleblair) [inline src/pages/docs/media/api-reference/index.js:1]: @darwin808 include Media in the valid urls for the API Reference page

**Review by giseleblair — COMMENTED** [review]
@darwin808 please include Media in the horizontal navigation dropdown selector

File: src/views/Docs/helper/index.js
Comment (giseleblair) [inline src/views/Docs/helper/index.js:1]: @darwin808 please clean up comments

**Review by giseleblair — APPROVED** [review]

**Review by giseleblair — DISMISSED** [review]

**Comment by giseleblair** [comment]
@darwin808 please review the conflicts and update the PR

**Comment by darwin808** [comment]
> @darwin808 please review the conflicts and update the PR

done

**Review by giseleblair — APPROVED** [review]

**Pull request merged**